model/prelude.sail

```diff
@@ -118,6 +118,17 @@ function bits_to_bool(x : bits(1)) -> bool = bool_bits(x)
 val to_bits : forall 'l, 'l >= 0.(int('l), int) -> bits('l)
 function to_bits (l, n) = get_slice_int(l, n, 0)
 
+val flip_bits : forall 'n, 'n >= 0. (int('n), bits('n)) -> bits('n)
+function flip_bits (num : int('n), x : bits('n)) -> bits('n) = {
```

Comment:
It seems like you are always flipping all bits? Can't you just xor with ones()?

If this function is needed with a partial flip, you can simplify that to an xor with `ones(n) @ zeros()`